gradle.properties

```diff
@@ -5,6 +5,7 @@ group=io.ballerina.lib
 version=1.0.0-SNAPSHOT
 api_inspectionlot_srvVersion=1.0.0-SNAPSHOT
 api_inspectionplan_srvVersion=1.0.0-SNAPSHOT
+api_masterinspcharacteristic_srvVersion =1.0.0-SNAPSHOT
```

Comment:
```suggestion
api_masterinspcharacteristic_srvVersion=1.0.0-SNAPSHOT
```

Reply:
Updated here [Update docs.json and gradle.properties](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/14/commits/b8c8fea8b5fc15e6761bed3bb7808449a7c7443d)